src/components/marginals/Footer/Footer.styles.js

```diff
@@ -0,0 +1,216 @@
+import styled from 'styled-components';
+import tw from 'twin.macro';
+
+const Grid = styled.div`
+  ${tw`
+    grid 
+    grid-cols-12
+    min-w-full
+    bg-secondary
+    p-4
+`}
+`;
+
+const SectionLogo = styled.div`
+  ${tw`
+  col-start-2
+  text-white
+  col-span-3
+  mid:col-start-1
+  text-left
+  md:hidden
+  grid
+  lg:grid-cols-6
+  grid-cols-9
+  
+`}
+`;
+
+const SectionLogoImg = styled.div`
+  ${tw`
+  inline-block
+  text-left
+`}
+`;
+
+const SectionLogoText = styled.div`
+  ${tw`
+  inline-block
+  m-0
+  lg:col-span-4
+  col-span-7
+  col-start-2
+  mx-1
+  lg:my-1
+
+`}
+`;
+
+const SectionIcon = styled.div`
+  ${tw`
+  col-start-8
+  col-span-4
+  mid:col-start-8
+  mid:col-span-5
+  text-color-secondary
+  
+  md:col-start-1
+  md:col-span-12
+  my-auto
+  mx-auto
+  
+`}
+`;
+
+const SectionIcons = styled.div`
+  ${tw`
+  text-white
+  text-right
+  md:text-center
+  grid
+  grid-cols-5
+  gap-4
+`}
+`;
+
+const SectionIconText = styled.div`
+  ${tw`
+  text-white
+  text-right
+  md:text-left
+  md:m-2
+`}
+`;
+
+const SectionCard1 = styled.div`
+  ${tw`
+  col-start-2
```

Comment:
Also, you haven't used grid here. It's more like you've hard-coded the dimensions of the card component by specifying `col-spam-4` and `row-span-2`

Where as, what you should have done instead is you could've used a Wrapper with display grid i.e:
`grid grid-cols-3` and then, inside the Wrapper, you could've mapped to the card contents returning the Card component.